interview-questions.md

```diff
@@ -8,50 +8,139 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: 
+    Parameters are placeholders when writing out function code. they are used to reference what argument will be passed in when the function is invoked. in the structure of a function, they are written in parentheses before the arrow notation. when called upon by the function, the function uses the arguments that are passed through in place of the parameters to log the result.
```

Comment:
Excellent work in this section!  You are really great at expanding these answers and being exceptionally thorough.  Great job!!